### DISCUSSION TITLE
Zoom

### DISCUSSION THREAD

**Discussion by roysinn** [liedekef/events-made-easy#110] [Q&A] [answered]
I'm starting to investigate if or how EME could be a gateway to Zoom registration. My group hosts a lot of Zoom sessions but we use EME for registration. We use EME for registration because it is great and Zoom isn't. But, our problem is reconciling EME registrations with Zoom attendance - i.e. people that register on EME as "John Doe" with the address "johndoe@mailservice.com" and then connect to the Zoom meeting as "Zontar" or whatever. I'm hoping that there is some Zoom API will allow EME to create a unique Zoom link for each EME registration. I would be most grateful for any suggestions.

**Comment by liedekef** [comment, marked as answer]
I don't know Zoom enough for API info (that is beyond EME). However EME provides hooks so you can script things after an event is entered or after someone registers. See the doc: https://www.e-dynamics.be/wordpress/category/documentation/12-hooks-and-filters/

**Reply by roysinn** [reply]
Franky - Thanks for the suggestion.
